prime-dotnet-webapi/Controllers/EnrolleeAgreementsController.cs

```diff
@@ -125,8 +125,8 @@ public async Task<ActionResult<Agreement>> GetAgreement(int enrolleeId, int agre
         [ProducesResponseType(StatusCodes.Status401Unauthorized)]
         [ProducesResponseType(StatusCodes.Status403Forbidden)]
         [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]
-        [ProducesResponseType(typeof(ApiResultResponse<EnrolleeProfileVersion>), StatusCodes.Status200OK)]
-        public async Task<ActionResult<EnrolleeProfileVersion>> GetEnrolmentForAgreement(int enrolleeId, int agreementId)
+        [ProducesResponseType(typeof(ApiResultResponse<Submission>), StatusCodes.Status200OK)]
+        public async Task<ActionResult<Submission>> GetEnrolmentForAgreement(int enrolleeId, int agreementId)
```

Comment:
This method's anme and URL don't really make sense any more. How about GetSubmissionForAgreement and change the URL to /submission ?